### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by jonasscheid**

### CODE REVIEW

**Comment by jonasscheid** [comment]
I think we need to remove the checks for python 3.6 since here since pandas >= 1.3.5 cannot be satisfied with that python version.. I am a bit suprised that we did not ran into the pandas problem beforehand tbh.. What do you think @christopher-mohr

**Comment by christopher-mohr** [comment]
> I think we need to remove the checks for python 3.6 since here since pandas >= 1.3.5 cannot be satisfied with that python version.. I am a bit suprised that we did not ran into the pandas problem beforehand tbh.. What do you think @christopher-mohr

I think we did not run into the problem since Pandas was not fixed and therfore it could be resolved right? Just thinking if there are any indications also for the nf-core pipeline when we basically remove Python `3.6` support...  🤔

**Comment by jonasscheid** [comment]
Hmm yes, so I think the problem occurs for pandas<1.3.5 and pandas dropped support for py36 [>1.2](https://thoth-station.ninja/j/pandas_py_36_drop.html)

Anyways. The current container for the nf-core pipeline uses py37, so should be fine I think..

**Comment by christopher-mohr** [comment]
> Hmm yes, so I think the problem occurs for pandas<1.3.5 and pandas dropped support for py36 [>1.2](https://thoth-station.ninja/j/pandas_py_36_drop.html)
> 
> Anyways. The current container for the nf-core pipeline uses py37, so should be fine..

Removed the tests for `develop` branch for now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,7 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Changed`
 
 - [#62](https://github.com/KohlbacherLab/epytope/pull/62) - Update the supported alleles of `syfpeithi` [#60](https://github.com/KohlbacherLab/epytope/issues/60)
-- Switch from [PyVCF](https://github.com/jamescasbon/PyVCF) to [PyVCF3](https://github.com/dridk/PyVCF3) as dependency
+
```
[the remark this answers is not in the document]
Comment (jonasscheid) [reply CHANGELOG.md:54]: Yup, thats why included it here directly

**Comment by jonasscheid** [comment]
@christopher-mohr like this?

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (christopher-mohr) [reply CHANGELOG.md:54]: In general, it's better to make sure that `develop` is up to date and the fork is based on the updated version.

**Review by christopher-mohr — APPROVED** [review]
Looks good in general. Not sure if we need it. Alternatively, merge `master` to `develop` and remove test from matrix there (maybe in fork).

**Pull request merged**